### ISSUE TITLE
Some LAS files are not rendered

### ISSUE THREAD

**Issue by SoftwareMechanic** [xeokit/xeokit-sdk#1358]
Some LAS files are not rendered, but in some cases there are no errors when model is loaded and the AABB is calculated correctly.
in other cases I receive an error -> `"this._state.flagsBuf is null"`

Here I attach one las file which is giving no errors and is not rendered:
[WeTransfer link](https://we.tl/t-LKfdJBmLdl)

Here I attach another las file which is giving an error when loading:
[WeTransfer link](https://we.tl/t-GUgX8Zvs0A)

Both links will last one week, I will resend you new links if needed.

In order to verify if these files are valid I tried open both this '.las' files on lidarview.com and I can see them.

 - OS: Windows 10
 - Browser: chrome, firefox

Please let me know if you need more Info about this issue.

**Comment by xeolabs** [comment]
Thanks - what version are these las files?

**Comment by SoftwareMechanic** [comment]
The first one should be 1.2, the second one I am not sure honestly.
I will ask to my coworker

**Comment by paireks** [comment]
Just for the record I could open both files using "las_IndoorScan" example without any errors: 

![2024-02-01_23h51_30](https://github.com/xeokit/xeokit-sdk/assets/47977819/681b051d-b292-451f-9414-c324eaed8fc5)

![2024-02-01_23h56_27](https://github.com/xeokit/xeokit-sdk/assets/47977819/c1f212e3-14c3-4985-9a1d-1ec8cddba23c)

The only thing I had to change was setting the camera at the end to be in a proper position, cause these scans can appear in totally different locations:
`sceneModel.on("loaded", () => {
        viewer.cameraFlight.jumpTo(sceneModel);
 });`

Tested on chrome and Windows 10.

**Comment by xeolabs** [comment]
Also, LASLoaderPlugin and convert2xkt oonly supports LAS/LAZ files up to LAS v1.3. They do not support LAS v1.4 files.

**Comment by SoftwareMechanic** [comment]
Thank you for these clarifications,
I am still having the same issue, but in the coming days I will retry also using the example provided.
Thank you!

**Comment by SoftwareMechanic** [comment]
Hello!
I tested with example as you suggested me and yes, it works within the example.
So I tried to upload the package.

But still, I have some las files which are visible and these which are loaded correctly but not rendered.

I tried using this code:

```
sceneModel.on("loaded", () => {
       console.log("Model loaded")
      
       this.viewer.cameraFlight.jumpTo(sceneModel); 

      this.modelsLoaded.push({id: sceneModel.id, sceneModel: sceneModel})
      //const aabb = this.viewer.scene.aabb         
      //this.viewer.cameraFlight.flyTo(aabb);
})
```

the event is called correctly (it works also with other las files and all other file formats that xeokit can read).
the las loader settings are the same and also the viewer.scene.pointsMaterial options are the same.

maybe the issue could be related to the fact I am using Next.js?

it's very weird that with other files I don't have this problem...
